src/templates/bet/mb_trading.html.tt

```diff
@@ -1,5 +1,11 @@
 <div class="center-text">
-    <img class="spinner" src="[% request.url_for('images/pages/binary-symbol-logo.svg') %]" alt="" />
+    <div class="barspinner dark">
+        <div class="rect1"></div>
+        <div class="rect2"></div>
+        <div class="rect3"></div>
+        <div class="rect4"></div>
+        <div class="rect5"></div>
+    </div>
```

Comment:
Is it possible to put this part in a separate template file, and just include it everywhere needed?